src/server/saved_objects/migrations/core/call_cluster.ts

```diff
@@ -34,10 +34,10 @@ export interface CallCluster {
   (path: 'indices.getAlias', opts: { name: string } & Ignorable): Promise<AliasResult | NotFound>;
   (path: 'indices.getMapping', opts: IndexOpts): Promise<MappingResult>;
   (path: 'indices.getSettings', opts: IndexOpts): Promise<IndexSettingsResult>;
-  (path: 'indices.putMapping', opts: PutMappingOpts): Promise<any>;
-  (path: 'indices.putTemplate', opts: PutTemplateOpts): Promise<any>;
   (path: 'indices.refresh', opts: IndexOpts): Promise<any>;
   (path: 'indices.updateAliases', opts: UpdateAliasesOpts): Promise<any>;
+  (path: 'indices.deleteTemplate', opts: { name: string }): Promise<any>;
+  (path: 'cat.templates', opts: { format: 'json'; name: string }): Promise<Array<{ name: string }>>;
```

Comment:
Maybe not necessarily in this PR, but you might consider consolidating all these type defs with the ones that are now provided in the Elasticsearch plugin. You can import them directly, something like: `import { CallCluster } from 'src/legacy/core_plugins/elasticsearch'` should work.

Reply:
Man... I 100% agree. It'd be nice if we could just drop these entirely, as they've been quite a pain.